src/server/Document/index.jsx

```diff
@@ -57,7 +57,7 @@ const renderDocument = async ({
 
   const app = extractCritical(
     renderToString(
-      <ChunkExtractorManager extractor={commonLoadableState}>
+      <ChunkExtractorManager extractor={commonLoadableState} css>
```

Comment:
There didn't seem to be any other way of fixing this https://github.com/bbc/simorgh/commit/3ed014646eeedf52ac8f57ff8a4c873f8016cde9